app/javascript/gobierto_data/webapp/components/sidebar/SidebarCategories.vue

```diff
@@ -23,19 +23,18 @@
               <a
                 class="gobierto-block-header--link"
                 @click.stop="e => selectAllCheckbox_TEMP({ ...e, filter })"
-              >{{
-                filter.isEverythingChecked ? labelNone : labelAll
-              }}</a>
+              >{{ checkIfEverythingChecked(filter.isEverythingChecked) }}</a>
```

Comment:
I don't get this: you're calling a function<sup>1</sup> to check if a boolean value (filter.isEverythingChecked) is true o false, in order to display one label or another

Why don't you simply use a `<template v-if="filter.isEverythingChecked">...`??


<sup>1</sup> Bad practice. You should never call functions in the template